frontend/shared/components/src/groups/components/ReduceablePriceInput.vue

```diff
@@ -40,6 +46,8 @@ const {enabled, financialSupportSettings} = useFinancialSupportSettings({
     group: computed(() => props.group),
```

Comment:
-> `useFinancialSupportSettings` aanpassen zodat 'enabled' altijd true is als app (useAppContext) === 'admin'